frontend-react/src/components/Admin/AdminReceiverDashboard.tsx

```diff
@@ -517,7 +501,6 @@ function MainRender(props: {
         }
         perDayElements = [];
         // </editor-fold>
-        keyOffset++;
```

Comment:
[tldr; consider renaming a few variables for clarity] removing this makes sense, I think (?), but trying to figure out why is making me more confused than I was before. I think the confusion comes down to what `key` and `currentEntry` represent, as that is somewhat obscured by the DataDictionary type. Each `key` is essentially a `receiverTimeSlotCheckId` or a `recieverCheckId` and each entry is a `timeSlotCheckResult`. Maybe it's just me, but I had forgotten this, and it really seemed like each key represented a receiver, so this was very confusing.

Reply:
Good feedback. Naming is hard and I always love feedback on better naming.

The bug was that offset inc was causing first time slot to be skipped.

<img width="170" alt="image" src="https://user-images.githubusercontent.com/74203452/182415384-6f6db88c-68c9-4011-95e5-b022588397fe.png">

Reply:
Part of the confusion around the key, keyOffset and currentEntry: I think code could be refactored to remove the key. Originally, I was assuming I could use the key to make sorting and lookups faster, but I think it just makes everything more complex.

Can I keep the name for the PR and see if I convert the key/value into just an ordered array of values, then I can use the index instead. So `data[key[keyOffset]]` -> `data[offset]`

Reply:
yeah, that's fine

Reply:
https://github.com/CDCgov/prime-reportstream/issues/6288